components/IngredientSelector/Navigator.tsx

```diff
@@ -144,8 +146,16 @@ export default function Navigator() {
   if (currentQuantity === null) {
     return null;
   }
-  const isWeightValue = currentQuantity?.quantity_default_weight !== undefined;
-  const updateStep = 1;
+
+  const isWeightValue = currentQuantity.quantity_default_weight !== undefined;
```

Comment:
to avoid the duplicated code with IngredientCard.tsx, maybe we could set "isWeightValue" and "step" in parseDataset.js, and then have a function currentQuantity.unitName()